src/Uno.UI.BindingHelper.Android/Uno/UI/UnoMotionHelper.java

```diff
@@ -133,14 +137,15 @@ private boolean dispatchMotionEventCore(Uno.UI.MotionTargetAdapter adapter, Moti
 		_currentMotionOriginalSource = null;
 
 		final boolean isDown = event.getAction() == MotionEvent.ACTION_DOWN;
-		if (isDown) {
+		final boolean isBeginOfSequence = isDown || event.getAction() == MotionEvent.ACTION_HOVER_ENTER;
```

Comment:
```suggestion
		final boolean isBeginningOfSequence = isDown || event.getAction() == MotionEvent.ACTION_HOVER_ENTER;
```

Reply:
Yes makes sense! 

But like the comment from Jérôme, I'll fix it in my next PR as I already did some changes in that file (and especially I changed this variable usage).